<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordEditorForm</class>
 <widget class="QMainWindow" name="WordEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zkanji - Edit word</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Editing word in dictionary:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dictImgLabel">
         <property name="text">
          <string notr="true">TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dictLabel">
         <property name="text">
          <string notr="true">English</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="kanjiLabel">
        <property name="text">
         <string>Written word:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QToolButton" name="kanjiButton">
            <property name="toolTip">
             <string>Write a Japanese character with mouse or pen</string>
            </property>
            <property name="icon">
             <iconset resource="Resources/resources.qrc">
              <normaloff>:/kanjipen.svg</normaloff>:/kanjipen.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ZKanaLineEdit" name="kanjiEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>5</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Written form of word, possibly containing kanji</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Word frequency:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ZLineEdit" name="freqEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Enter relative word frequency. Higher numbers mean higher frequency of use</string>
             </property>
             <property name="text">
              <string notr="true">0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="kanaLabel">
        <property name="text">
         <string>Kana reading:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ZKanaLineEdit" name="kanaEdit">
        <property name="toolTip">
         <string>Kana reading of the word. Make sure this matches the written form</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="wordAttribLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Word attributes:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wordAttribWidget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Definitions:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>198</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="cloneButton">
        <property name="toolTip">
         <string>Duplicate selected definition. The new definition will be added at the end of the list</string>
        </property>
        <property name="text">
         <string>Clone</string>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="delButton">
        <property name="toolTip">
         <string>Remove selected definition</string>
        </property>
        <property name="icon">
         <iconset resource="Resources/resources.qrc">
          <normaloff>:/delmark.svg</normaloff>:/delmark.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ZDictionaryEditListView" name="defTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ZStatusBar" name="listStatus"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Definition text, different translations in their own line:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="WordAttribWidget" name="attribWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Definition attributes:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QPlainTextEdit" name="defTextEdit">
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Enter each translation for this definition on a new line.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZLineEdit</class>
   <extends>QLineEdit</extends>
   <header>zlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ZStatusBar</class>
   <extends>QStatusBar</extends>
   <header>zstatusbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZKanaLineEdit</class>
   <extends>QLineEdit</extends>
   <header>zkanalineedit.h</header>
  </customwidget>
  <customwidget>
   <class>WordAttribWidget</class>
   <extends>QWidget</extends>
   <header>wordattribwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZDictionaryEditListView</class>
   <extends>QTableView</extends>
   <header>wordeditorform.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
